ApollosProject/apollos-apps
# fix: Removes gradient top on default card

## DESCRIPTION
This PR removes the gradient top overlay on all default cards.

### What does this PR do, or why is it needed?
This PR removes `overlayType` and `overlayColor` on the default card and horizontal default card.

### What design trade-offs/decisions were made?

### How do I test this PR?
Use `storybook` to navigate to default cards and horizontal default cards to confirm no gradient.

![Simulator Screen Shot - iPhone 11 Pro - 2020-09-25 at 12 39 58](https://user-images.githubusercontent.com/52924611/94293608-84360c80-ff2c-11ea-8370-6169bdcefb46.png)


### What automated tests did you write?

## TODO:


## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
